add: How to deploy a container in Amazon ECS with instances (#17)
toledompm/devops-culture: docs/infrastructure/ecs.md

## Amazon ECS clusters

> An Amazon ECS cluster is a logical grouping of **tasks** or **services**. A cluster is also a logical grouping of capacity providers. And also allows to create multiple clusters to keep resources separate.

Reviewer: ```suggestion
An Amazon ECS cluster is a logical grouping of **tasks** or **services**, as well as a logical grouping of capacity providers. Multiple clusters can be created to keep resources separate.
```
